src/core/Akka.Remote/Transport/DotNetty/DotNettyTransportSettings.cs

```diff
@@ -87,7 +87,7 @@ public static DotNettyTransportSettings Create(Config config)
                 serverSocketWorkerPoolSize: ComputeWorkerPoolSize(config.GetConfig("server-socket-worker-pool")),
                 clientSocketWorkerPoolSize: ComputeWorkerPoolSize(config.GetConfig("client-socket-worker-pool")),
                 maxFrameSize: ToNullableInt(config.GetByteSize("maximum-frame-size", null)) ?? 128000,
-                ssl: config.HasPath("ssl") ? SslSettings.Create(config.GetConfig("ssl")) : SslSettings.Empty,
+                ssl: config.HasPath("ssl") && config.GetBoolean("enable-ssl", false) ? SslSettings.Create(config.GetConfig("ssl")) : SslSettings.Empty,
```

Comment:
`enableSsl && config.HasPath("ssl")` can be achieved by moving line [#81](https://github.com/akkadotnet/akka.net/blob/2b1d88894d455040bda8d20d70f048ba6e7330c8/src/core/Akka.Remote/Transport/DotNetty/DotNettyTransportSettings.cs#L81) outside the return, by otherwise LGTM.